site/gatsby-site/src/components/forms/SubmissionWizard/StepThree.js

```diff
@@ -155,6 +155,10 @@ const FormDetails = ({
 }) => {
   const { t } = useTranslation(['submit']);
 
+  const { t: tVariants } = useTranslation(['variants']);
+
+  const { t: tTranslations } = useTranslation();
```

Comment:
I think we should change these two lines to this line `const { t } = useTranslation(['translation', 'variants']);`
On file `site/gatsby-site/src/components/discover/Actions.js` we have an example of this